File: root_VS2012_TemplateBase/programs/VB/Samples/WebApp_sample/ProjectX_sample/App_Code/sample/Common/TestReturnValue.vb
'**********************************************************************************
'* フレームワーク・テストクラス
'**********************************************************************************

'**********************************************************************************
'* クラス名        ：TestReturnValue
'* クラス日本語名  ：テスト用の戻り値クラス
'*
'* 作成日時        ：－
'* 作成者          ：sas 生技
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'*
'**********************************************************************************

' ベースクラス
Imports Touryo.Infrastructure.Business.Common

Namespace MyType
    ''' <summary>
    ''' TestReturnValueの概要の説明です
    ''' </summary>
    Public Class TestReturnValue
        Inherits MyReturnValue
        ''' <summary>汎用エリア</summary>
        Public Obj As Object

        ''' <summary>ShipperID</summary>
        Public ShipperID As Integer

        ''' <summary>CompanyName</summary>
        Public CompanyName As String

        ''' <summary>Phone</summary>
        Public Phone As String
    End Class
End Namespace

File: root/programs/C#/Samples/WinServiceProject/WinServiceProject/Codes/Common/TestReturnValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

// ベースクラス
using Touryo.Infrastructure.Business.Common;

namespace WinServiceProject.Codes.Common
{
    public class TestReturnValue : MyReturnValue
    {
        /// <summary>汎用エリア</summary>
        public object Obj;

        /// <summary>UserId</summary>
        public string UserId;

        /// <summary>RegistrationDateTime</summary>
        public DateTime RegistrationDateTime;

        /// <summary> ExecutionStartDateTime;</summary>
        public DateTime ExecutionStartDateTime;

        /// <summary> NumberOfRetries;</summary>
        public int NumberOfRetries;

        /// <summary>ProgressRate</summary>
        public int ProgressRate;

        /// <summary> Status;</summary>
        public string Status;

        /// <summary> Command;</summary>
        public string Command;

        /// <summary>ReservedArea</summary>
        public string ReservedArea;
    }
}